src/client/adapter.ts

```diff
@@ -179,6 +179,10 @@ export const convexAdapter = (
           if (!("runMutation" in ctx)) {
             throw new Error("ctx is not a mutation ctx");
           }
+          if (data.where?.[0] && !data.where?.[0]?.operator) {
+            data.where![0].operator = "eq";
+          }
```

Comment:
The adapter should use eq as default, curious if this change is addressing a failure?